modules/simple-payments/simple-payments.php

```diff
@@ -377,5 +404,173 @@ function setup_cpts() {
 		register_post_type( self::$post_type_product, $product_args );
 	}
 
+	/**
+	 * Format a price for display
+	 *
+	 * Largely taken from WordPress.com Store_Price class
+	 *
+	 * The currency array will have the shape:
+	 *   format  => string sprintf format with placeholders `%1$s`: Symbol `%2$s`: Price.
+	 *   symbol  => string Symbol string
+	 *   desc    => string Text description of currency
+	 *   decimal => int    Number of decimal places
```

Comment:
FYI I confirmed that all the decimals match with [Calypso side](https://github.com/Automattic/wp-calypso/blob/37830bdc69f2bddfac1746e5bb1b9a120510d509/client/lib/format-currency/currencies.js)